pkg/api/meta/builds.go

```diff
@@ -46,6 +46,12 @@ func (m *buildSpecMutator) Mutate(fn ImageReferenceMutateFunc) field.ErrorList {
 		if err := fn(&s.From); err != nil {
 			errs = append(errs, fieldErrorOrInternal(err, m.path.Child("strategy", "sourceStrategy", "from", "name")))
 		}
+		if s.RuntimeImage != nil {
```

Comment:
Why aren't we removing this in 3.7?

Reply:
good question.  i'll do a follow up to rip it out.  I intentionally didn't bother adding any tests for it in this PR :)